crypto/pem/pem_lib.c

```diff
@@ -93,6 +86,13 @@ int PEM_def_callback(char *buf, int num, int w, void *key)
         return (i);
     }
 
+#if defined(OPENSSL_NO_STDIO) || defined(OPENSSL_NO_UI)
+    /*
+     * We should not ever call the default callback routine from windows.
```

Comment:
The comment here is a little odd...it doesn't seem to match up with the ifdef guards...i.e. I would expect it to say something about no-stdio and no-ui